timeless/restaurants/table_shapes/views.py

```diff
@@ -64,3 +64,7 @@ def delete(id):
     table_shape = models.TableShape.query.get(id)
     DB.session.delete(table_shape)
     return redirect(url_for("table_shape.list"))
+
+
+def order_by(query, params):
```

Comment:
@Vatavuk Since this will be used in most viewes, lets move this function, for example, to `templates/views.py`? wdyt?

Reply:
@vryazanov 👍